api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/summary/SummaryHelpersTests.kt

```diff
@@ -0,0 +1,201 @@
+                "maxPool2D(MaxPool2D)                 ::[None, 8, 8, 64]  ::0              ::conv2D_2           ",
+                "...............................................................................................",
+                "conv2D_4(Conv2D)                     ::[None, 8, 8, 64]  ::36928          ::maxPool2D          ",
+                "...............................................................................................",
+                "conv2D_5(Conv2D)                     ::[None, 8, 8, 64]  ::36928          ::conv2D_4           ",
+                "...............................................................................................",
+                "add(Add)                             ::[None, 8, 8, 64]  ::0              ::conv2D_5           ",
+                "                                     ::                  ::               ::maxPool2D          ",
+                "...............................................................................................",
+                "conv2D_6(Conv2D)                     ::[None, 8, 8, 64]  ::36928          ::add                ",
+                "...............................................................................................",
+                "conv2D_7(Conv2D)                     ::[None, 8, 8, 64]  ::36928          ::conv2D_6           ",
+                "...............................................................................................",
+                "add_1(Add)                           ::[None, 8, 8, 64]  ::0              ::conv2D_7           ",
+                "                                     ::                  ::               ::add                ",
+                "...............................................................................................",
+                "conv2D_8(Conv2D)                     ::[None, 6, 6, 64]  ::36928          ::add_1              ",
+                "...............................................................................................",
+                "globalAvgPool2D(GlobalAvgPool2D)     ::[None, 64]        ::0              ::conv2D_8           ",
+                "...............................................................................................",
+                "dense_1(Dense)                       ::[None, 256]       ::16640          ::globalAvgPool2D    ",
+                "...............................................................................................",
+                "dense_2(Dense)                       ::[None, 10]        ::2570           ::dense_1            ",
+                "...............................................................................................",
+                ":::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::",
+                "Total trainable params: 222666",
+                "Total frozen params: 911333",
```

Comment:
Could the number of frozen params be odd? we need a double check here

Reply:
Does this model have frozen params? No it has not frozen params. Frozen params are params of the frozen layer. There is no frozen layers

Reply:
Oh, I was confused, because the purpose of this test was to test the ModelSummary and could contain an unrealistic number of frozen params, better to use numbers 222666->2570, 911333->220 096, and 1133999->220 096 to avoid misunderstanding for the future supporters of these tests

Reply:
changed number to real ones